monitor.py

```diff
@@ -517,6 +517,13 @@ def parse_entry(self, line: bytes) -> MutableMapping[str, Any]:  # noqa: C901, C
                 self.live = True  # First event in 3.0
 
             elif event_type == 'loadgame':
+                # Odyssey Release Update 5
```

Comment:
I don't see how we could possible have a `loadgame` without a `fileheader` - we're not pulling journals from the CAPI (which doesn't have `fileheader` and that is why the fields have been added to `loadgame` as well).

So this seems un-necessary.

Reply:
Put there as a "just in case", that said, it can be easily removed now that its been DRY-ed